packages/vscode-extension/src/debugging/CDPBreakpoint.ts

```diff
@@ -0,0 +1,133 @@
+import { Breakpoint, Source } from "@vscode/debugadapter";
+import { SourceMapController } from "./SourceMapsController";
+import { Logger } from "../Logger";
+import { CDPSession } from "./CDPSession";
+
+export class CDPBreakpoint extends Breakpoint {
+  public readonly line: number;
+  public readonly column: number | undefined;
+  private _id: number | undefined;
+  private executingQueue: boolean = false;
+  private cdpCommunicationQueue: Array<() => Promise<void>> = [];
+
+  constructor(
+    private cdpSession: CDPSession,
+    private sourceMapController: SourceMapController,
+    verified: boolean,
+    line: number,
```

Comment:
feels very weird that we pass line number and column but we don't pass the filename. It feels like line number and column makes it much more strict. There's no way breakpoint could move between files keeping its line and column, no?

Reply:
I actually didn't think about it before but the name of the file could change  during fast refresh.
After some initial test it's seems to me that this scenario break fast refresh, because it does not update the files that import the old file. 

The current behavior seems to be that the bundle is not actually updated, which means that the old breakpoint is still there but it is not visible for the user, that would suggest that the break point itself should contain it's sourceFile. 

I am not sure about any of this tho, maybe in this scenario we should induce "Reload JS" ourselves to improve developer experience?  I this scenario we would still like to use old breakpoints but with a new source file. 

Can we leave it as is for now and I'll adres it in a follow up when I'll figure how to best handle the scenario of fast refreshing after changing the file name?